webclient/app/src/app/features/apps/entities/relationshipAccordion/RelationshipAccordion.jsx

```diff
@@ -64,6 +67,22 @@ function RelationshipAccordion(props) {
                                         ))}
                                     </Select>
                                 </FormControl>
+                                <FormGroup>
```

Comment:
Ist hier der Abstand richtig (Nach oben)? Bitte gegebenenfalls noch <Stack> einfügen

Reply:
In der UI sind die Abstände richtig